README.md

```diff
@@ -16,7 +18,7 @@ OpenMC, Matplotlib, NumPy, PySide2
 This package is most easily installed by running
 
 ```console
-pip install .
+pip install openmc-plotter
 ```
 
 from the top directory in the repository.
```

Comment:
Looks like we can remove this line now?

Reply:
I have removed ```pip install .``` and replaced it with ```pip install openmc-plotter``` in this PR should this be done differently

Reply:
I think @pshriwise's point was about the line below: "from the top directory in the repository" no longer makes sense.

Reply:
ah got you, I was distracted by the colors